include/aws/http/connection_manager.h

```diff
@@ -161,12 +161,12 @@ int aws_http_connection_manager_release_connection(
     struct aws_http_connection *connection);
 
 /**
- * Fetch the current manager metric from connection manager.
+ * Fetch the current manager metrics from connection manager.
  */
 AWS_HTTP_API
-void aws_http_connection_manager_fetch_metric(
-    struct aws_http_connection_manager *manager,
-    struct aws_http_manager_metric *out_metric);
+void aws_http_connection_manager_fetch_metrics(
+    const struct aws_http_connection_manager *manager,
+    struct aws_http_manager_metrics *out_metric);
```

Comment:
trivial: ~out_metric~ -> out_metrics

some compilers issue warnings if the param name is different between the .h and .c file